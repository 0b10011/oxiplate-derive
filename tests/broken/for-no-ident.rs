use oxiplate_derive::Oxiplate;

#[derive(Oxiplate)]
#[oxiplate_inline = "{%- for in numbers -%}No ident present{% endfor %}"]
struct Data {
    numbers: Vec<i32>,
}

fn main() {
    print!(
        "{}",
        Data {
            numbers: vec![19, 89],
        }
    );
}
