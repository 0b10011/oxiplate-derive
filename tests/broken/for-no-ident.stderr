error: Expected 'in'
 --> tests/broken/for-no-ident.rs:4:33
  |
4 | #[oxiplate_inline = "{%- for in numbers -%}No ident present{% endfor %}"]
  |                                 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
